preflight-checks/src/checks/02-check-port-and-healthcheck-route.js

```diff
@@ -0,0 +1,59 @@
+import chalk from 'chalk';
+import packageJson from '../package';
+const execa = require( 'execa' );
+const waait = require( 'waait' );
+const fetch = require( 'isomorphic-fetch' );
+
+const CACHE_HEALTHCHECK_ROUTE = '/cache-healthcheck?';
+const chalkNpmStart = chalk.yellow( 'npm start' );
+const chalkPORT = chalk.yellow( 'PORT' );
+const chalkHealthCheckRoute = chalk.yellow( CACHE_HEALTHCHECK_ROUTE );
+const chalkVIPGo = chalk.yellow( '@automattic/vip-go' );
+
+module.exports = {
+	name: `Building the app and running ${ chalkNpmStart }...`,
+	excerpt: `Checking if your app accepts a ${ chalkPORT } and responds to ${ chalkHealthCheckRoute }`,
+	run: async () => {
+		const scripts = packageJson.scripts;
+		const PORT = Math.floor( Math.random() * ( 4000 - 3000 ) + 3000 ); // Get a PORT between 3000 and 4000
+		let buildingCommand = 'npm run build';
```

Comment:
Do we need to run `npm install --production` first?

Reply:
oh yes, we need to run `npm install` before, thank you for catching this!

Reply:
should be fixed with this https://github.com/Automattic/vip-go-node/pull/118/commits/dd43b7f148351fc4bac4f636e6e9a3dcb1eaff53